x/stakeibc/keeper/hooks.go

```diff
@@ -80,6 +80,9 @@ func (k Keeper) BeforeEpochStart(ctx sdk.Context, epochInfo epochstypes.EpochInf
 		if epochNumber%StrideEpochsPerDayEpoch == 0 {
 			k.RebalanceAllHostZones(ctx)
 		}
+
+		// Transfers in and out of tokens for hostZones which have community pools
```

Comment:
What's the reason behind running it at this point in the hooks.go flow? Why not earlier / later? I don't have well formed thoughts on where it should be run, mostly curious.

Reply:
The ordering of these functions doesn’t matter since they’re all emitted in the same block

but there’s a different question as to whether we should submit these hourly or stride-epochly

pros of hourly is it completes the cycle faster, but con is that we’re then issuing a ton of pointless ICQs because of how infrequent this feature is used

I think my vote is probably to change it to hourly and add a feature flag, but I don't have a strong opinion. Curious to hear your thoughts!

Reply:
I agree, hourly is a lot more ICQ to an address which will be empty 99.9% of the time. 3 hours vs 18 hours isn't that different of an experience for the community pool.  The only thing which would change the UX is if we decide to keep records in the future so that we can add query endpoints to report how many funds are in flight at any moment and what stage they are in.